doc/030_preparing_a_new_repo.rst

```diff
@@ -239,6 +239,10 @@ For an S3-compatible server that is not Amazon (like Minio, see below),
 or is only available via HTTP, you can specify the URL to the server
 like this: ``s3:http://server:port/bucket_name``.
 
+.. note:: Currently, restic only supports defining the `path-style URLs <https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#access-bucket-intro>`__.           Virtual-hosted–style URLs, where the bucket name is in the 
```

Comment:
I'd change the wording such that it says that restic expects the URLs to be in path-style and that URLs in virtual-hosted-style must be converted. That could look like this:
```
.. note:: restic expects `path-style URLs <https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#access-bucket-intro>`__
          like for example ````s3.us-west-2.amazonaws.com/bucket_name``.
          Virtual-hosted–style URLs like ``bucket_name.s3.us-west-2.amazonaws.com``,
          where the bucket name is part of the hostname are not supported. These must
          be converted to path-style URLs instead, for example ``s3.us-west-2.amazonaws.com/bucket_name``.
```

Reply:
I've updated it!